### ISSUE TITLE
Can not set the learner for permutation.importance filter

### ISSUE THREAD

**Issue by studerus** [mlr-org/mlr#814]
I understand that the permutation.importance filter requires a learner to be specified, like in this example:
```r
filterFeatures(pid.task, method = 'permutation.importance', abs = 5, learner = 'classif.logreg')
```

However, if I want to use to fuse a base learner with this filter, I cannot specify the learner for the permutation.importance because that argument name is already used for the base learner. I want to do something like this
```r
wrapped.lrn <- makeFilterWrapper(learner = 'classif.randomForest', fw.abs = 3,
   fw.method = 'permutation.importance', fw.learner = 'classif.logreg')
train(wrapped.lrn, pid.task)
```

**Comment by zmjones** [comment]
so we just need to change the name of the learner argument for the learner used to measure importance?

**Comment by studerus** [comment]
Yes, I think that would solve the problem.

**Comment by zmjones** [comment]
ok i will do that and push the change in a bit

**Comment by berndbischl** [comment]
Hi, sorry a few comments:

1) @studerus posted dome code that did not work. Hence, we should add a test in the PR.

2) I dont like the name change to fw.learner. that somehow suggests it is an arg of the FilterWrapper (as we use fw.* as a conventio there)

3) This is probably a separate issue, be seem to COMPLETELY lack docs for the details of the filters and their signatures?

**Comment by zmjones** [comment]
Yes that is correct that there are no docs. I definitely think we need them. How do you think we should do that? Do you have an alternative arg name in mind? What about `f.learner`?

**Comment by berndbischl** [comment]
> Yes that is correct that there are no docs. I definitely think we need them. How do you think we should do that?

Create a separate page ?filters. add rox docs in front of every filter in R code, direct them to ?filters with rox tag "@rdname"?

> you have an alternative arg name in mind? What about f.learner?

how about imp.learner? we calc the importance with that learner?

**Comment by berndbischl** [comment]
we where not DONE here!

sorry, I kinda mistakenly commented in the issuem should have done in the PR!
@larskotthoff 
(what you should have seen when you should have reviewed the state of the ISSUE thread AS WELL)

lets pls fix this now ASAP so we dont run into problems

**Comment by larskotthoff** [comment]
Do you want me to revert?

**Comment by berndbischl** [comment]
wait for zachs response.

IMHO he could simply iterate over.

**Comment by berndbischl** [comment]
wait what he say. revert would be cleaner for the repo state

**Comment by zmjones** [comment]
yea just revert it and i will modify my branch

ok so `imp.learner` is fine

for docs i am not sure what you mean.

so i'd create a new file filters.R which has generic documentation for all filters, and then in front of each filter function i'd add docs for additional args, and specify that this should be associated with filters.Rd? 

i can describe many of the filter methods but i definitely don't know how all of them work...

**Comment by berndbischl** [comment]
@zmjones 
If you want to you could fix the renaming like we suggested here, in a sep PR, and then do the docs. 
Just as you like.
But if you fix the renaming pls also add a test

**Comment by zmjones** [comment]
ok will do. i have it all in one branch now

**Comment by berndbischl** [comment]
was fixed by zach in PR #820
